Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MaxTextLength.swift
+//  Mobile Test Runner for Jama
+//
+//  Created by PSU1 on 7/5/17.
+//  Copyright © 2017 Jaca. All rights reserved.
+//
+
+import Foundation
```

Comment:
@benleelawrence @JasonRitz I am not sure if you guys saw this comment. What are your thoughts on it. I meant "utils folder" and "ViewControllers" folder in above comment.